commands/FBFlickerCommands.py

```diff
@@ -62,20 +63,14 @@ def run(self, arguments, options):
 
 class FlickerWalker:
   def __init__(self, startView):
-    self.setCurrentView(startView)
-
-    self.handler = inputHelpers.FBInputHandler(lldb.debugger, self.inputCallback)
-    self.handler.start()
+    self.setCurrentView(startView, None)
 
   def run(self):
-    while self.handler.isValid():
-      self.flicker()
-
-  def flicker(self):
-    viewHelpers.setViewHidden(self.currentView, True)
-    time.sleep(0.1)
-    viewHelpers.setViewHidden(self.currentView, False)
-    time.sleep(0.3)
+    self.keepRunning = True
+    lldb.debugger.SetAsync (True)
```

Comment:
Can you comment on why this is necessary? And, remove the space after `SetAsync`?